src/Controller/PagesController.php

```diff
@@ -31,14 +31,13 @@ class PagesController extends AppController
     /**
      * Displays a view
      *
+     * @param array ...$path
```

Comment:
That was my that was my first intention but PHPStorm complained about it so I updated but I agree with you that string will better fit.
I'll update that as well as the travis build matrix where I forgot to remove 5.5